pkg/issuer/venafi/setup.go

```diff
@@ -36,7 +35,7 @@ func (v *Venafi) Setup(ctx context.Context) error {
 
 	err = client.Ping()
 	if err != nil {
-		klog.Infof("Issuer could not connect to endpoint with provided credentials. Issuer failed to connect to endpoint\n")
+		v.log.V(logf.WarnLevel).Info("Issuer could not connect to endpoint with provided credentials. Issuer failed to connect to endpoint")
```

Comment:
Shouldn't we log the error here *and* in the condition message below.
On the other hand, the error also gets wrapped and returned, so I wonder if that version of the error is also logged?